sections/Discussion/process-reflections.tex

```diff
@@ -0,0 +1,36 @@
+These sprints served as a guideline for when to have sprint retrospectives and enact changes in the process.
+Each retrospective would cause slight alterations to the process.
+\\\\
+Initially, tasks were estimated based on planning poker.
+However, we found it difficult to properly estimate a lot of the tasks, as this project concerned itself with an area of expertise we were not familiar with.
+This led to the inclusion of an Essence based version of estimating in sprint 1.
+This version of estimation was also scrapped, as described in \autoref{sec:sprint2-conclusion}, due to it seeming biased towards shorter tasks and estimation not being accurate.
+This led to the method of simply having a group-based discussion about the DoD of a task, and ensuring everyone understood it.
+This DoD based task assignment worked well for the final sprints.
+It avoided spending unnecessary time on planning poker on top of discussing tasks and made the developers more productive.
+\\\\
+Another aspect improved by the retrospectives was the stand-up meetings.
+At the beginning of the project these tended to be unfocused and not provide the necessary information, and were not conducted daily.
+This led to the introduction of a certain set of questions, defined in \autoref{sec:sprint2-conclusion}, which helped to ensure the developers stayed focused and provided the necessary information to facilitate productivity.
+Along with the questions, the stand-up meetings were changed to be conducted daily, which further facilitated the developers staying focused.
+\\\\
+\subsection{Evaluation of Essence}
+One of the main things that was used from Essence was the roles.
+While Essence includes things like potential and filters for the pre-project, these were not introduced until the pre-project phase of the semester had passed, and was deemed less meaningful to use at these stages.\\
+Additionally, since not all group members took the software innovation class where Essence was presented, it took a lot of work to relay the information since the remaining group members would essentially have to learn the curriculum of another course on top of the ones they were already following.
+\\\\
+Our experience with the Essence roles was that it was quite similar to the roles present in the well-known Scrum approach.
+The challenger, in Essence, has a similar role as a product owner, the anchor as the scrum master, and the responder corresponds to being a member of the development team.
+The primary addition that Essence has to roles is the addition of the child role.
+This role was supposed to be a fluctuating role which allowed group members to come with feedback and encourage divergent thinking.\\
+However, since a lot of the pre-project parts of Essence were skipped, the child role did not make a lot of sense and quickly ended up becoming a joke to suggest silly ideas rather than a useful tool.
```

Comment:
synes a lof of the pre-project parts er lidt blown ift vi aldrig rigtgit introducerer dem. Kan bedre lige at vi formulerer det som many of the proe-project parts